src/doc.d

```diff
@@ -1122,7 +1126,7 @@ extern (C++) void toDocBuffer(Dsymbol s, OutBuffer* buf, Scope* sc)
             // emit constraints if declaration is a templated declaration
             if (td && td.constraint)
             {
-                buf.writestring(" if (");
+                buf.writestring("$(DDOC_CONSTRAINT ");
```

Comment:
Not sure I follow @JakobOvrum. Do you mean in the default macros just make `DDOC_CONSTRAINT = $0` then add that change to std.ddoc? Does adding it there override the default macros for dlang.org or something?